src/components/aws/ListEC2Instances.vue

```diff
@@ -173,7 +173,7 @@
       createSnapshot: function(row) {
         if (this.snapshotDescription == "" || this.snapshotVolume == "") {
             this.$toast.open({
-                message: 'Bitte fülle alle Felder aus',
+                message: 'Please fill out all the blanks',
```

Comment:
Please fill out all THE blanks 
or
Please fill out all blanks

Reply:
Ich finde beides nicht sehr passend. Wieso blanks? 
```suggestion
                message: 'Please fill out all the inputs',
```

Reply:
Bei einem Formular nennt man die Felder blanks, in welche man etwas eingeben kann.
oder nicht?

Reply:
Ich kenne den Ausdruck nicht. Aber mir fällt nicht gerade viel besseres ein.

Reply:
https://www.linguee.de/englisch-deutsch/uebersetzung/please+fill+in+the+blank.html

https://www.linguee.de/englisch-deutsch/uebersetzung/please+fill+out+all+required+fields.html

Was sagst du zu diesen Optionen